client/i18n/locales/english/translations.json

```diff
@@ -102,13 +102,14 @@
       "contains invalid characters": "Username \"{{username}}\" contains invalid characters",
       "is too short": "Username \"{{username}}\" is too short",
       "is a reserved error code": "Username \"{{username}}\" is a reserved error code",
+      "must be lowercase": "Username \"{{username}}\" must be lowercase",
       "unavailable": "Username not available",
       "validating": "Validating username...",
       "available": "Username is available",
       "change": "Please note, changing your username will also change the URL to your profile and your certifications."
     },
     "labels": {
-      "username": "Username",
+      "username": "Username (lowercase only)",
```

Comment:
I'm wondering whether we need to make this change. IMO, just throwing the validation warning should be enough.

Let's get some input on this

Reply:
I agree.  It seems unnecessary to single out case when we validate for a few things and the validation message is hard to miss.

Reply:
@thecodingaviator, Yep, you are right about validation warning is enough. At first, I was not sure about changing the username label since they were discussing it in the issue. That is why I would like to hear from them.

@ojeytonwilliams Yeah.

I will change the username label back.